internal/host/validator.go

```diff
@@ -202,7 +202,7 @@ func (v *validator) printHasMinMemory(c *validationContext, status validationSta
 	case ValidationSuccess:
 		return "Sufficient minimum RAM"
 	case ValidationFailure:
-		return fmt.Sprintf("Require at least %d GiB RAM, found only %d GiB", v.hwValidatorCfg.MinRamGib,
+		return fmt.Sprintf("The host is not eligible to participate in Openshift Cluster because the minimum required RAM for any role is %d GiB, found only %d GiB", v.hwValidatorCfg.MinRamGib,
```

Comment:
"does not", or "do not"?

Reply:
Isn't similar to the previous message that was previously reject because it is similar to the "min-memory-for-role" validation?

Reply:
@andybraren ^

Reply:
Sorry, I'm not sure where to find the previous message that was rejected (I'm not super familiar with this repo) - could you provide a link to it?

Reply:
There are 2 validations for RAM that are for different purposes. The first (what we are discussing in this PR), is to verify that the cluster is installable for any of the roles.  If this validation fails, then the user has to know that the host cannot be installed.
There is another validation that checks that host has enough memory for the selected role.  In this case the user can change the role to fix this validation failure.
We would like to let the user distinguish between the two by a clear enough message for each.
This is the issue: https://issues.redhat.com/browse/OCPBUGSM-18248

Reply:
@andybraren ^

Reply:
@ori-amizur What message was rejected?  By who?  Where?

Reply:
@avishayt look at the issue.  Since there are 2 RAM validations that have different purpose, we decided to explain them better in order to distinguish between the cases.

Reply:
We can also decide not to change, but the issue needs to be closed.

Reply:
@ronniel1 What do you think?

Reply:
OK I understand now, thanks
@andybraren We want the message to explicitly say that the host doesn't meet requirements for any role and therefore can't participate in the cluster, to differentiate from the validation where the host doesn't meet requirements for a specific role.  Can you please suggest different messages?

Reply:
@avishayt @andybraren How can we continue?